<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataManagementView</class>
 <widget class="QDialog" name="DataManagementView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/tango-actions-32px/images/tango-icon-theme/32x32/actions/document-properties.png</normaloff>:/tango-actions-32px/images/tango-icon-theme/32x32/actions/document-properties.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_Experimental">
       <property name="title">
        <string>Files Containing &amp;Experimental Data</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="listWidget_Experimental"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>58</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_BrowseExperimental">
              <property name="text">
               <string>&amp;Browse to Add...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_RemoveExperimental">
              <property name="text">
               <string>&amp;Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Simulation">
       <property name="title">
        <string>Files Containing &amp;Simulation Data</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QListWidget" name="listWidget_Simulation"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>58</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_BrowseSimulation">
              <property name="text">
               <string>&amp;Browse to Add...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_RemoveSimulation">
              <property name="text">
               <string>&amp;Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>428</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox_OkCancelResetApply">
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_BrowseExperimental</tabstop>
  <tabstop>listWidget_Experimental</tabstop>
  <tabstop>pushButton_RemoveExperimental</tabstop>
  <tabstop>pushButton_BrowseSimulation</tabstop>
  <tabstop>listWidget_Simulation</tabstop>
  <tabstop>pushButton_RemoveSimulation</tabstop>
  <tabstop>buttonBox_OkCancelResetApply</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox_OkCancelResetApply</sender>
   <signal>accepted()</signal>
   <receiver>DataManagementView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_OkCancelResetApply</sender>
   <signal>rejected()</signal>
   <receiver>DataManagementView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
